ONSdigital/alpha-eq-survey-runner
# Digeq 94 more cassandra changes

**What**
Storing the cassandra session object globally as it handles connection pooling for us and we want to reuse it. Chris and I had lots of fun trying to use the flask 'g' and 'current_app' variables, as suggested by the documentation, but they are only available per request and oddly not for the duration of the application as documented. Global variables seems to be the only way to store state in flask between requests, which seems a little odd. 

Also created an init script for setting up the cassandra session keyspaces and tables.

**How to test**
1. Start the application and check that the keyspace is created
2. Run through the debug survey and check it all still works

**Who can review**
Probably best @dhilton does, as Dan has reviewed the other related pull requests.
